### PR TITLE
[ru] Translate docs/concepts/containers/ - RU K8s docs

### PR DESCRIPTION
**Pull request by kirkonru**
Russian localization for [docs/concepts/containers/*](https://github.com/kubernetes/website/tree/main/content/en/docs/concepts/containers)

### CODE REVIEW

**Comment by Sea-n** [comment]
In this repo, we tend to squash commits into one commit in most cases.

Could you do it in your local git client?

**Comment by kirkonru** [comment]
@Sea-n sorry for the delayed response – there were some technical difficulties on my side. Commits are squashed and ready for review. Thank you for your suggestion!

**Comment by divya-mohan0209** [comment]
Folks, I don't mean to impose/overstep. However this has been pending approval/review since 6 months. 
@kubernetes/sig-docs-ru-owners , @kubernetes/sig-docs-ru-reviews : PTAL.

**Comment by divya-mohan0209** [comment]
I'm aware that the cadence is generally slower for localizations. However, if the lack of approvers/reviewers merits a wider discussion in he localization subproject meeting, please do bring it up. Alternately, if the subproject lead/docs leads can help out anyway, please do let us know.

File: content/ru/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+```
+
+При наличии файла учетных данных Docker можно импортировать их как {{< glossary_tooltip text="Secret'ы" term_id="secret" >}} Kubernetes вместо команды, приведенной выше.
+ 
+В разделе [Создание Secret'а на основе существующих учетных данных Docker](/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials) рассказывается, как это можно сделать.
+
+Это особенно удобно в случае нескольких приватных реестров контейнеров, так как `kubectl create secret docker-registry` создает Secret, который работает только с одним приватным реестром.
+
+{{< note >}}
+Pod'ы могут работать только с Secret'ами в собственном пространстве имен, поэтому этот процесс необходимо повторить для каждого пространства имен.
```
Comment (shurup) [inline content/ru/docs/concepts/containers/images.md:224]: Just minimizing a repetition here.
```suggestion
Pod'ы могут работать только с Secret'ами в собственном пространстве имен, поэтому данный процесс необходимо повторить для каждого пространства имен.
```

**Review by shurup — COMMENTED** [review]
All looks perfect to me! @kirkonru: here are just a few minor corrections for your consideration.

**Comment by shurup** [comment]
@divya-mohan0209 thank you very much for your reminder! This PR was really huge, and that was one of the reasons to postpone it so long. We already agreed to do new translations in smaller chunks, so we would (hopefully) move better.

@Arhell would you be able to have a look at this, too?

**Comment by kirkonru** [comment]
@shurup hello, this one is ready for merging as well. :sunglasses:

**Comment by shurup** [comment]
Thanks, @kirkonru, all looks good now!
/lgtm
/approve

**Pull request merged**